back/src/Guide/Api/Types.hs

```diff
@@ -93,7 +96,7 @@ data CategorySite route = CategorySite
       Summary "Get contents of a category"
       :> ErrorResponse 404 "Category not found"
       :> "category"
-      :> Capture "id" (Uid Category)
+      :> Capture "category id" (Uid Category)
```

Comment:
By convention, path variables shouldn't have spaces. You can use `"categoryId"`, `"itemId"` and so on.